store/cachekv/store.go

```diff
@@ -280,6 +281,7 @@ const (
 
 // Constructs a slice of dirty items, to use w/ memIterator.
 func (store *Store) dirtyItems(start, end []byte) {
+	const THRESHOLD = 1024
```

Comment:
can we move this outside the method and give it a better name?

Reply:
Also explain the reasoning behind 1024 in a godoc comment :)

Reply:
Sure, though 1024 pre-dates this change and apparently is a magic number: https://github.com/cosmos/cosmos-sdk/pull/10026#discussion_r697965203

It's re-used in this PR since it made sense (to me) that the minimum number of elements removed matches the threshold to sort.

Reply:
@alexanderbez moved and renamed.  Since it's magic and documented where it's used, I haven't added a godoc comment.  Open to suggestions.